packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -143,10 +143,20 @@ const App = (props) => {
         l10nConfig: site.l10n
     })
 
+    // Workaround for infinite loop if default site locale translation file is missing
+    const isDefaultTranslation = new RegExp(
+        `/static/translations/compiled/${site.l10n.defaultLocale}`
```

Comment:
```suggestion
        `^/static/translations/compiled/${site.l10n.defaultLocale}$`
```
[minor] using `^` and `$` makes the RegExp more performant.

Reply:
Added `.json$ to the end of the regex to address performance.

I opted to not include the `^` since the full `location.pathname` in my local is `/mobify/bundle/development/static/translations/compiled/en-GB.json` and adding the `/mobify/bundle/{your environment}` part doesn't feel correct.